tests/dailytest/case.go

```diff
@@ -106,18 +106,35 @@ var case2Clean = []string{`
 }
 
 var case3 = []string{`
-CREATE TABLE a(id INT PRIMARY KEY, a1 INT);
+CREATE TABLE binlog_case3(id INT PRIMARY KEY, a1 INT);
 `,
 	`
-INSERT INTO a(id, a1) VALUES(1,1),(2,1);
+INSERT INTO binlog_case3(id, a1) VALUES(1,1),(2,1);
 `,
 	`
-ALTER TABLE a ADD UNIQUE INDEX aidx(a1);
+ALTER TABLE binlog_case3 ADD UNIQUE INDEX aidx(a1);
 `,
 }
 
 var case3Clean = []string{
-	`DROP TABLE a`,
+	`DROP TABLE binlog_case3`,
+}
+
+// Test issue: TOOL-1346
+var case4 = []string{`
```

Comment:
To further improve readability, you can rename the case to tell the maintainer what it's about, for example, `caseTblWithGeneratedCol` and `caseSetBitColumn` would tell us far more than just `case1, 2, 3, 4`.

Reply:
revised